### PR TITLE
different start state

### PR DESCRIPTION
**Pull request by mikramarc**

### CODE REVIEW

**Review by beatrizleon — APPROVED** [review]

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 1]
```diff
@@ -187,20 +187,25 @@ def move_to_joint_value_target(self, joint_states, wait=True,
...
 
-    def plan_to_joint_value_target(self, joint_states, angle_degrees=False):
+    def plan_to_joint_value_target(self, joint_states, angle_degrees=False, custom_start_state=None):
         """
         Set target of the robot's links and plans.
         @param joint_states - dictionary with joint name and value. It can
         contain only joints values of which need to be changed.
         @param angle_degrees - are joint_states in degrees or not
+        @param custom_start_state - specify if plannig start state different
+        then the current state
```
Comment (beatrizleon) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]:
```suggestion
        than the current state
```
same for all other comments. Can it fit in one line with the previous?

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 2]
```diff
@@ -187,20 +187,25 @@ def move_to_joint_value_target(self, joint_states, wait=True,
...
         self._move_group_commander.go(wait=wait)
 
-    def plan_to_joint_value_target(self, joint_states, angle_degrees=False):
+    def plan_to_joint_value_target(self, joint_states, angle_degrees=False, custom_start_state=None):
         """
         Set target of the robot's links and plans.
         @param joint_states - dictionary with joint name and value. It can
         contain only joints values of which need to be changed.
         @param angle_degrees - are joint_states in degrees or not
+        @param custom_start_state - specify if plannig start state different
```
Comment (beatrizleon) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]:
```suggestion
        @param custom_start_state - specify a start state different than the current state
```

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py (hunk 1 above)
Comment (mikramarc) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: fixed in 8a85ebb

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py (hunk 2 above)
Comment (mikramarc) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: fixed in 8a85ebb

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 3]
```diff
@@ -193,8 +193,7 @@ def plan_to_joint_value_target(self, joint_states, angle_degrees=False, custom_s
         @param joint_states - dictionary with joint name and value. It can
         contain only joints values of which need to be changed.
         @param angle_degrees - are joint_states in degrees or not
-        @param custom_start_state - specify if plannig start state different
-        then the current state
+        @param custom_start_state - specify if plannig start state different then the current state
```
Comment (beatrizleon) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: it is still "then" instead of "than" :)

Comment (mikramarc) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: dammit, sorry, fixed in 18af4e8

Comment (mikramarc) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: and in e0049b8, c4a5809

**Pull request merged**